monitor/monitor.go

```diff
@@ -216,15 +216,15 @@ func (m *Monitor) monitorDPoSValidatorChange() {
 				// self init sync if add validator
 				if validatorChange.EthAddr == m.ethClient.Address {
 					log.Infof("%s. Init my own validator.", logmsg)
-					m.isValidator = true
+					m.bond()
```

Comment:
`setBonded(true)`, so that it's clear this is only setting the field, not actually bonding on-chain.